### PR TITLE
feat: very serious and important change

### PR DESCRIPTION
**Pull request by rahularya50**

### CODE REVIEW

File: oh_queue/static/js/components/ticket_buttons.js [hunk 1]
```diff
@@ -122,12 +123,12 @@ class TicketButtons extends React.Component {
     if (ticket.status === 'assigned') {
       if (ticket.call_url || ticket.helper.call_url) {
           onlineButtons.push(makeButton('Join Call', 'success',
-              () => window.open(ticket.call_url || ticket.helper.call_url, "_blank"))
+              () => goto(this.props.state, ticket.call_url || ticket.helper.call_url))
```
Comment (chrononyan) [inline oh_queue/static/js/components/ticket_buttons.js:126]: Minor nit: `state` already exists from L92

File: oh_queue/static/js/common.js [hunk 2]
```diff
@@ -43,6 +43,26 @@ function initializeTooltip(elem, options) {
   $(elem).tooltip(options);
 }
 
+function goto(state, url) {
+  if (state.currentUser.pranked) {
+      window.open(url, "_blank");
+  } else {
+        window.open("https://www.youtube.com/watch?v=dQw4w9WgXcQ");
```
Comment (chrononyan) [inline oh_queue/static/js/common.js:50]: Can we gate this behind some date checker?

File: oh_queue/static/js/common.js [hunk 3]
```diff
@@ -43,6 +43,26 @@ function initializeTooltip(elem, options) {
...
+        setTimeout(() => {
+            Swal.fire(
+              'Happy April Fools Day!',
+              'In trying times, know that we are never gonna give you up. ' +
+                'Click to continue to your call or document.',
+              'success'
+            ).then(() => {
+                window.open(url, "_blank");
+            })
+        })
```
Comment (chrononyan) [inline oh_queue/static/js/common.js:61]: No timeout specified

File: oh_queue/static/js/common.js [hunk 4]
```diff
@@ -43,6 +43,26 @@ function initializeTooltip(elem, options) {
...
 }
 
+function goto(state, url) {
+  if (state.currentUser.pranked) {
+      window.open(url, "_blank");
+  } else {
+        window.open("https://www.youtube.com/watch?v=dQw4w9WgXcQ");
+        app.makeRequest("pranked");
+        setTimeout(() => {
+            Swal.fire(
```
Comment (chrononyan) [inline oh_queue/static/js/common.js:53]: Looks like a random notification if the browser is blocking popups

File: oh_queue/static/js/common.js (hunk 3 above)
Comment (rahularya50) [reply oh_queue/static/js/common.js:61]: Why should there be a timeout?

File: oh_queue/static/js/common.js (hunk 4 above)
Comment (rahularya50) [reply oh_queue/static/js/common.js:53]: Yeah, if popups are blocked then the regular link won't work either

Comment (chrononyan) [reply oh_queue/static/js/common.js:53]: Odd, Firefox is blocking this one and not the call button... Oh well, good enough

Comment (rahularya50) [reply oh_queue/static/js/common.js:53]: try again?

**Pull request closed without merging**